src/content/docs/style-guide/structure/styleguide-markup-indentation.mdx

```diff
@@ -0,0 +1,324 @@
+---
+title: "Markdown indentation"
+tags:
+  - Basic style guide
+  - Style guide quick reference
+---
+
+import indentSnippetsInLists from 'images/indent-snippets-in-lists.png'
+
+import indentSnippetsInListsTwo from 'images/indent-snippets-in-lists-2.png'
+
+import indentLeftJustified from 'images/indent-left-justified.png'
+
+import indentExtraLineFeed from 'images/indent-extra-line-feed.png'
+
+import topology1 from 'images/topology-1.png'
+
+import topology2 from 'images/topology-2.png'
+
+import childBulletList from 'images/child-bullet-list.png'
+
+import childNumberedList from 'images/child-numbered-list.png'
+
+import fourSpaceIndentChild from 'images/four-space-indent-child.png'
+
+import threeSpaceIndentChild from 'images/three-space-indent-child.png'
+
+import basicListIndentationLeft from 'images/basic-list-indentation-left.png'
+
+import basicListIndentationTwo from 'images/basic-list-indentation-two.png'
+
+import nestingBulletListOne from 'images/nested-bullet-lists-1.png'
+
+import nestingBulletListTwo from 'images/nested-bullet-lists-2.png'
+
+import consecutiveListCollapserOne from 'images/indentation-consec-lists-in-collapser-1.png'
+
+import consecutiveListCollapserTwo from 'images/indentation-consec-lists-in-collapser-2.png'
+
+import imageIndentationOne from 'images/image-indentation-1.png'
+
+When you're writing in Markdown, indentation can make your mark-up more readable. Indentation can also affect the file's readability when it's rendered in HTML. This doc covers how to handle parent (first-level), child (second-level), and grandchild (third-level) lists. We'll also demonstrate how to insert spaces to incorporate code blocks, images, and nested lists within a set of procedures.
+
+## List indentation [#lists]
+
+Lists are a great way to explain procedures (numbered steps) or simply itemize items (bullet lists). While single-level lists are pretty straightforward, when you start nesting them, you can run into indentation issues. Take a look at these list topics in this section:
+
+* [First-level lists (parents)](#first-level)
```

Comment:
Love that you have this here since it won't show up on the right table of contents!

Reply:
I hope we eventually can get our developers to add h3 so we can see them in the right nav!